beaverhabits/frontend/components.py

```diff
@@ -35,9 +35,12 @@ def compat_menu(name: str, callback: Callable):
     return ui.menu_item(name, callback).props("dense").classes("items-center")
 
 
-def menu_icon_button(icon_name: str, click: Optional[Callable] = None) -> Button:
+def menu_icon_button(icon_name: str, click: Optional[Callable] = None, tooltip: Optional[bool]|str = False) -> Button:
```

Comment:
Thanks for contributing to the repo, I just had a chance to review it :p 

The type annotation should be: 
```
tooltip: Optional[str] = None
# or
tooltip: str | None = None
```

Reply:
@daya0576  Thank you for the review; I have updated the code with your suggestion and cleaned up the way of returning the button